From 1f6246e9ca52c7800345791c12e38bf9fb32280b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Fri, 31 Oct 2025 15:46:29 +0100
Subject: [PATCH 037/133] std.Target: bump opencl/nvcl max version to 3.0.19
X-Developer-Signature: v=1; a=openpgp-sha256; l=719; i=xtex@astrafall.org;
 h=from:subject; bh=drVPJvLYGTIqmimDq2lMfhpx+YWo8AVlkfaxo34Cwfk=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn7cv9lfdPMmthizbGwMYJDU2P9o3qVaSfaReR2Cac1P
 zwgyCrWMZGFQYyLwVJMkaXIsMGbVSedX3RZuSzMHFYmkCHSIg0MDAwMLAx8uYl5pUY6Rnqm2oZ6
 hoY6RjpGDFycAjDV8rwM/3P38GZ5WOu88Uh6tipK7rZ0wY3GdW9L5i6NWXVp+7MVtwIZfrP5LTB
 K02D4FvVMTt+gWk9x0bM3ot3778ikSTCUXfZ3YQQA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit d677086bcdb1497775e5b84b00fcaa0e7fab895b)
---
 lib/std/Target.zig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/std/Target.zig b/lib/std/Target.zig
index 2e42450ae31b..5d3cc6258f51 100644
--- a/lib/std/Target.zig
+++ b/lib/std/Target.zig
@@ -635,7 +635,7 @@ pub const Os = struct {
                 => .{
                     .semver = .{
                         .min = .{ .major = 2, .minor = 2, .patch = 0 },
-                        .max = .{ .major = 3, .minor = 0, .patch = 17 },
+                        .max = .{ .major = 3, .minor = 0, .patch = 19 },
                     },
                 },
                 .opengl => .{
-- 
2.52.0

